src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -432,7 +433,23 @@ export class DebugService implements IDebugService {
 							nls.localize('debugTypeMissing', "Missing property 'type' for the chosen launch configuration.");
 					}
 
-					await this.showError(message);
+					let actionList: IAction[] = [];
+					let action = {
```

Comment:
Instead of creating a full action like this inline, I suggest the following
`const action = new Action(....)`
That way you will not have to specify all these parameters

Reply:
Example https://github.com/microsoft/vscode/blob/1cba068f10d5175c286f43aa6ad33985aee39abd/src/vs/workbench/contrib/debug/browser/breakpointEditorContribution.ts#L295

Reply:
Thanks for the suggestion and example! I modified to `actionList.push(new Action(...`